lbz/events/api.py

```diff
@@ -115,3 +116,19 @@ def _create_eb_entry(self, new_event: BaseEvent) -> PutEventsRequestEntryTypeDef
             "Resources": self._resources,
             "Source": self._source,
         }
+
+
+def event_emitter(function: Callable) -> Callable:
+    """Decorator that makes function an emitter - automatically sends pending events on success"""
+
+    @wraps(function)
+    def wrapped(*args: Any, **kwargs: Any) -> Any:
```

Comment:
As I mentioned in the description, the same logic we have implemented in the [EventAwareResource](https://github.com/pdyba/lambdalizator/blob/1d785bb0f3c9b54898321428337df565c6ea6507/lbz/resource.py#L247) class but there we depend on the response status code.